### PR TITLE
doc: update OCR image information

### PR DESCRIPTION
**Pull request by tbarnes-us**
Monica, Derek, and I are collaborating on changes to reflect the latest information about CPU/GA images, refine discussions about when they are patched, and their license terms.

### CODE REVIEW

File: documentation/staging/content/userguide/platforms/environments.md [hunk 1]
```diff
@@ -82,27 +82,78 @@ For more information, see the [Fusion Middleware Licensing Information User Manu
 
 #### Oracle Linux and WebLogic Server images
 
+{{% notice warning %}}
+Oracle strongly recommends using dated Critical Patch Update (CPU) images from the Oracle Container Registry (OCR),
```
Comment (rosemarymarano) [inline documentation/staging/content/userguide/platforms/environments.md:86]: Start the note/warning with, "For production deployments, Oracle recommends..."

File: documentation/staging/content/userguide/platforms/environments.md [hunk 2]
```diff
@@ -82,27 +82,78 @@ For more information, see the [Fusion Middleware Licensing Information User Manu
...
+  - Includes critical security fixes for Oracle Linux, Java, and Oracle WebLogic Server.
+  - Suitable for production use.
+
+- General Availability (GA) images.
+  - Located in OCR repositories "middleware/weblogic" and "middleware/fmw-infrastructure".
+  - Updated quarterly.
+  - Includes latest updates for Oracle Linux, and Java, but _not_ for Oracle WebLogic Server.
+  - GA images are subject to [Oracle Technology Network (OTN) Developer License Terms](https://www.oracle.com/downloads/licenses/standard-license.html), 
+    which include, but are not limited to:
+     - Must only be used for the purpose of developing, testing, prototyping, and demonstrating applications.
```
Comment (rosemarymarano) [inline documentation/staging/content/userguide/platforms/environments.md:121]: Must only be used -> Must be used only

**Review by rosemarymarano — COMMENTED** [review]
minor edits

File: documentation/staging/content/userguide/platforms/environments.md [hunk 3]
```diff
@@ -146,14 +150,76 @@ Example CPU images:
 | container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol7-211124 | JDK 8u311, Oracle Linux 7u9, and Oracle WebLogic Server 12.2.1.4 generic distribution October 2021 CPU |
 | container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol7 | Represents latest JDK 8, latest Oracle Linux 7, and GA Oracle WebLogic Server 12.2.1.4 generic distribution CPU |
 
+You may have noticed that the image tags may include keywords like `generic`, `slim`, etc.
+This reflects the type of WebLogic install in the image. There are multiple types,
```
Comment (rosemarymarano) [inline documentation/staging/content/userguide/platforms/environments.md:154]: WebLogic install -> WebLogic installation

File: documentation/staging/content/userguide/platforms/environments.md [hunk 4]
```diff
@@ -146,14 +150,76 @@ Example CPU images:
...
+- `.../weblogic...:...generic...`:
+  - The WebLogic generic image is supported for development and production deployment
+    of WebLogic configurations using Docker.
+  - Contains the same binaries as those installed by the WebLogic generic installer.
+- `.../weblogic...:...slim...`:
+  - The WebLogic slim image is supported for development and production deployment
+    of WebLogic configurations using Docker.
+  - In order to reduce image size,
+    it contains a subset of the binaries included in the WebLogic generic image:
+    - The WebLogic Console, WebLogic examples, WebLogic clients, Maven plug-ins,
```
Comment (rosemarymarano) [inline documentation/staging/content/userguide/platforms/environments.md:165]: WebLogic Console -> WebLogic Server Administration Console

File: documentation/staging/content/userguide/platforms/environments.md [hunk 5]
```diff
@@ -146,14 +150,76 @@ Example CPU images:
...
+  where `TAG` is one of `12.2.1.3`, `12.2.1.4`, `14.1.1.0-11`, or `14.1.1.0-8`.
+  These are created with the generic installer, 
+  Oracle Linux 7-slim, and JDK8 
+  except for `14.1.1.0-11` (which has JDK11).
+- The tag for an undated image may not 
+  specify its WebLogic installer type, 
+  in which case you can assume it is `generic`.
+  Otherwise, such images may have
+  a tag that includes the string `slim` or `dev`
+  string, in which case they have the
```
Comment (rosemarymarano) [inline documentation/staging/content/userguide/platforms/environments.md:208]: includes the string `slim` or `dev` string, -> remove one instance of "string"

**Review by rosemarymarano — COMMENTED** [review]
Text edits

**Review by rosemarymarano — COMMENTED** [review]
Please let me know when this doc is **completely** done and I will review it.

File: documentation/staging/content/userguide/base-images/_index.md [hunk 6]
```diff
@@ -568,28 +539,37 @@ _Important_:
...
+
+  - Creates a new WebLogic image layered on
+    an existing WebLogic image (specified in WIT's `--fromImage` parameter)
+    - Note that if you specify the `--pull` parameter for WIT,
+      and the `--fromImage` refers to an image in a repository,
+      and the repository image is newer than the locally cached version of the image,
+      then the command will download the repository image
+      to the local docker cache and use it
+      instead of using the outdated local image.
+  - This option can be used to generate a domain home or a model home in the new image using WDT.
```
Comment (rosemarymarano) [inline documentation/staging/content/userguide/base-images/_index.md:572]: This option can be used -> You can use this option

**Review by rosemarymarano — COMMENTED** [review]
Some minor edits, I will certainly need to review the document/documents in their entirety before approving them. Note that I am skeptical of reproducing WIT documentation here (in the WKO docs) but am aware that it's usefulness can and should be explained in this context. Bottom line is: I don't want us to be responsible for updating WIT documentation in two doc sets.

**Review by rosemarymarano — COMMENTED** [review]
@tbarnes-us I incorporated my edits but there were a few points that I needed you to address, hence the review comments. **HOWEVER**, there is a bigger picture that you, me, @mriccell, @ddsharpe, and perhaps, @robertpatrick should address as to whether the entirety of this information belongs in this one document. I will set up a meeting with all of us for tomorrow.

File: documentation/staging/content/userguide/base-images/_index.md [hunk 7]
```diff
@@ -1,238 +1,1146 @@
...
+
+Here are some example WebLogic Server Oracle Container Repository (OCR) images,
+where the names are abbreviated to omit their `container-registry.oracle.com/middleware/` prefix:
+
+| Abbreviated Name | Descripton |
+|-|-|
+|`weblogic:12.2.1.4`|GA image with latest JDK 8, latest Oracle Linux 7, and the GA Oracle WebLogic Server 12.2.1.4 generic distribution. Note that this image has no date stamp, so it can change over time with potential updates to JDK 8 and Oracle Linux 7.|
+|`weblogic:12.2.1.4-YYMMDD`|GA image with JDK 8, Oracle Linux 7, and the GA Oracle WebLogic Server 12.2.1.4 generic distribution for the given date.|
+|`weblogic_cpu:12.2.1.4-generic-jdk8-ol7`|CPU image with latest JDK 8, latest Oracle Linux 7, and GA Oracle WebLogic Server 12.2.1.4 generic distribution CPU. Note that this image has no date stamp, so it can change over time with potential updates to JDK 8, to Oracle Linux 7, and to the latest CPU.|
+|`weblogic_cpu:12.2.1.4-generic-jdk8-ol7-211124`|CPU image with JDK 8u311, Oracle Linux 7u9, and the Oracle WebLogic Server 12.2.1.4 generic distribution October 2021 CPU.|
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md:177]: Yah - granted it's a bit confusing - but @mriccell confirmed this is expected when we were wrote this portion.

File: documentation/staging/content/userguide/platforms/environments.md (hunk 1 above)
Comment (tbarnes-us) [reply documentation/staging/content/userguide/platforms/environments.md]: @mriccell Can you take a look and decide? IIRC the previous guidance was that the note applies to all deployments - but it is just more strongly recommended for production deployments.

File: documentation/staging/content/userguide/platforms/environments.md [hunk 8]
```diff
@@ -89,6 +94,16 @@ for production deployments.
 General Availabity (GA) images are not licensable or suitable for production use.
 {{% /notice %}}
 
+{{% notice note %}}
+All of the OCR images that are discussed in this section are built using
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/platforms/environments.md]: Outdated comment.

File: documentation/staging/content/userguide/base-images/_index.md [hunk 9]
```diff
@@ -1,238 +1,1146 @@
...
+
+   **Note**: This step is only needed once for each image name (not the tag level).
+   For example, if you accept the terms for the "weblogic_cpu"
+   link in the "middleware" repository, then
+   the acceptance applies to all versions of WebLogic CPU images.
 
-```shell
-$ docker login container-registry.oracle.com
-```
+1. Log Docker in to the Oracle Container Registry. For example,
```
[the remark this answers is not in the document]
Comment (robertpatrick) [reply documentation/staging/content/userguide/base-images/_index.md:230]: Perform a Docker login to the Oracle Container Registry?

File: documentation/staging/content/userguide/platforms/environments.md (hunk 3 above)
Comment (tbarnes-us) [reply documentation/staging/content/userguide/platforms/environments.md]: now "type of WebLogic distribution installed in the image's Oracle Home." (text now moved to images doc)

File: documentation/staging/content/userguide/platforms/environments.md [hunk 10]
```diff
@@ -146,14 +150,76 @@ Example CPU images:
...
+    - WebLogic examples and Console help files have been removed.
+    - All binaries that remain included are the same as those in the WebLogic generic image.
+  - This image type is primarily intended to provide a Docker image
+    that is consistent with the WebLogic "quick installers" intended for development only.
+    Production WebLogic domains should use the WebLogic generic or WebLogic slim images.
+- `.../fmw-infrastructure...:...`:
+  - The Fusion Middleware (FMW) Infrastructure image is supported for
+    development and production deployment of FMW configurations using Docker.
+  - It contains the same binaries as those installed by the WebLogic generic installer
+    and adds Fusion Middleware Control and Java Required Files (JRF)
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/platforms/environments.md]: Already done. (Text has been moved to images doc)

File: documentation/staging/content/userguide/platforms/environments.md (hunk 5 above)
Comment (tbarnes-us) [reply documentation/staging/content/userguide/platforms/environments.md]: no longer applicable (and text now moved to image doc)

File: documentation/staging/content/faq/newbie.md [hunk 11]
```diff
@@ -115,9 +115,19 @@ For more information, see the [Domain life cycle]({{< relref "/userguide/managin
 
 #### Patching and Upgrades
 
-**Q:** Patching: rolling upgrades, handling of one-off-patches and overlays, CPUs, and such.
+**Q:** How can I get a patched Oracle container image for running in Kubernetes (one-off patches, overlays, CPUs, etc.)?
 
-**A:** For relevant information, see [Domain images]({{< relref "/userguide/base-images/_index.md" >}}), [Rolling restarts]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting#overview" >}}), and [CI/CD considerations]({{< relref "/userguide/cicd/_index.md" >}}).
+**A:** Download pre-patched images from the Oracle Container Registry or use WebLogic Image Tool to create your own.
+For additional information on WebLogic container images on Oracle Container registry, see [Obtain images from the Oracle Container Registry](userguide/base-images/#obtain-images-from-the-oracle-container-registry).
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/faq/newbie.md]: already done

File: documentation/staging/content/faq/newbie.md [hunk 12]
```diff
@@ -115,9 +115,19 @@ For more information, see the [Domain life cycle]({{< relref "/userguide/managin
 
 #### Patching and Upgrades
 
-**Q:** Patching: rolling upgrades, handling of one-off-patches and overlays, CPUs, and such.
+**Q:** How can I get a patched Oracle container image for running in Kubernetes (one-off patches, overlays, CPUs, etc.)?
 
-**A:** For relevant information, see [Domain images]({{< relref "/userguide/base-images/_index.md" >}}), [Rolling restarts]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting#overview" >}}), and [CI/CD considerations]({{< relref "/userguide/cicd/_index.md" >}}).
+**A:** Download pre-patched images from the Oracle Container Registry or use WebLogic Image Tool to create your own.
+For additional information on WebLogic container images on Oracle Container registry, see [Obtain images from the Oracle Container Registry](userguide/base-images/#obtain-images-from-the-oracle-container-registry). 
+For additional information on creating new container images with WebLogic Image Tool, see [Patch WebLogic Server images](userguide/base-images/#create-a-custom-image-with-patches-applied).
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/faq/newbie.md]: already done

File: documentation/staging/content/userguide/base-images/_index.md [hunk 13]
```diff
@@ -43,76 +43,56 @@ description: "Create, obtain, or inspect images for WebLogic Server or Fusion Mi
...
     * [Create a custom image with patches applied](#create-a-custom-image-with-patches-applied)
     * [Auxiliary images]({{< relref "/userguide/managing-domains/model-in-image/auxiliary-images.md" >}})
 
 * Model in Image domains that _do not_ leverage auxiliary images:
-  * This approach is for Model in Image type domains where
-    the WebLogic Deploy Tooling install and potentially also model files
-    are supplied in the same image that includes WebLogic Server or Fusion Middleware Infrastructure binaries.
+  * This is a single container image approach that groups the Oracle binaries, domain configuration, and application binaries into a single image.
+  * The Oracle Home binaries are provided by the OCR or custom image.
+  * The image containing the Oracle Home must be extended with the WebLogic Deploy Tooling install, the domain configuration as WebLogic Deploy Tooling models, and the application binaries.
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md]: not applicable (this entire bullet has been removed)

File: documentation/staging/content/userguide/base-images/_index.md [hunk 14]
```diff
@@ -43,76 +43,56 @@ description: "Create, obtain, or inspect images for WebLogic Server or Fusion Mi
...
-Customers can also use WIT to build their own WebLogic Server or Fusion Middleware Infrastructure images
-(with the latest Oracle Linux images, Java updates, and WebLogic Server patches),
-apply one-off patches to existing OCR images,
-or overlay their own files and applications on top of an OCR image.
-See [Contents](#contents) for information about using this tool
-to create custom WebLogic Server or Fusion Middleware Infrastructure images for the WebLogic Kubernetes Operator.
+All of the OCR images that are described in this section are built using the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool) (WIT). Customers can also use the WIT to build their own WebLogic Server or Fusion Middleware Infrastructure images (with the latest Oracle Linux images, Java updates, and WebLogic Server patches), apply one-off patches to existing OCR images, or overlay their own files and applications on top of an OCR image. See [creating your own custom image](##create-a-custom-image-with-patches-applied) for information about using this tool to create custom WebLogic Server or Fusion Middleware Infrastructure images for the WebLogic Kubernetes Operator.
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md]: already done

File: documentation/staging/content/userguide/base-images/_index.md [hunk 15]
```diff
@@ -122,31 +102,22 @@ See the following sections for information about OCR image names and accessing O
...
 
 {{% notice warning %}}
-The latest Oracle Container Registry (OCR) **GA images**
-include the latest security patches for Oracle Linux and Java,
-and do _not_ include the latest security patches for WebLogic Server.
-Oracle strongly recommends using images with the latest security patches,
-such as OCR Critical Patch Updates (CPU) images or custom generated images.
-See [Ensure you are using recently patched images]({{< relref "/userguide/base-images/_index.md#ensure-you-are-using-recently-patched-images" >}}).
+The _WebLogic Server images_ and _Fusion Middleware Infrastructure images_ (**GA images**) on OCR are updated quarterly to include the latest security patches for Oracle Linux and Oracle Java, but do _not_ include the latest security patches for WebLogic Server or Fusion Middleware Infrastructure. Oracle strongly recommends using images with all the latest security patches, such as the Critical Patch Updates (**CPU images**) images provided quarterly on OCR or custom generated images using WIT with `--recommendedPatches`.  See [Ensure you are using recently patched images]({{< relref "/userguide/base-images/_index.md#ensure-you-are-using-recently-patched-images" >}}).
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md]: already done

File: documentation/staging/content/userguide/base-images/_index.md [hunk 16]
```diff
@@ -1,238 +1,1146 @@
...
+    - [Create a custom base image](#create-a-custom-base-image)
+    - [Create a custom image with your domain inside the image](#create-a-custom-image-with-your-domain-inside-the-image)
+    - [Create a custom image with your model inside the image](#create-a-custom-image-with-your-model-inside-the-image)
+  - [Apply patched images to a running domain](#apply-patched-images-to-a-running-domain)
+    - [Domain in PV](#domain-in-pv)
+    - [Model in Image with auxiliary images](#model-in-image-with-auxiliary-images)
+    - [Model in Image without auxiliary images](#model-in-image-without-auxiliary-images)
+    - [Domain in Image](#domain-in-image)
+
+### Understanding WebLogic images
```
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md:38]: Moving overview and making it a top main section, and replacing this section with a short intro.

File: documentation/staging/content/userguide/base-images/_index.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md]: @tbarnes-us The only comment left for which I need you to clarify. Thanks!

[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md]: Already addressed, but feel free to word smith or change it to Robert's suggested wording. 

I changed the wording to:
```
1. Provide Docker with credentials for accessing
   the Oracle Container Registry. For example,
   the following command will prompt for your SSO credentials:

   ```shell
   $ docker login container-registry.oracle.com
   ```
```

File: documentation/staging/content/userguide/base-images/_index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (tbarnes-us) [reply documentation/staging/content/userguide/base-images/_index.md:189]: @robertpatrick : Not in this case. Dated images do not change over time.

@rosemarymarano  : Or ask @mriccell  for a better example -- if I recall correctly, she said there was something a bit messy about that particular quarter's naming.

[the remark this answers is not in the document]
Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:189]: @tbarnes-us ok, will do.

**Pull request merged**